### ISSUE TITLE
Make the sync-url flag respect iliasignore

### ISSUE THREAD

**Issue by funnym0nk3y** [FliegendeWurst/KIT-ILIAS-downloader#38]
It seems to me that the `.iliasignore` file is not respected when using the `--sync-url` flag. I use the flag for syncing a course before the lecture on my notebook instead of crawling all other courses. Thus, I'd like it to respect excluded folders from the `.iliasignore`.

**Comment by FliegendeWurst** [comment]
The `.iliasignore` file is not ignored. Do you have an additional `.iliasignore` file for the course?

For example, if you usually download all courses into `./ILIAS` and want to download a specific course in `./ILIAS/Analysis I`, the downloader will certainly read `./ILIAS/Analysis I/.iliasignore`. Did you expect the downloader to consider `./ILIAS/.iliasignore`?

**Comment by funnym0nk3y** [comment]
Exactly, I did expect it to use `./ILIAS/.iliasignore`, even though I invoked the command for a specific course folder.

**Comment by FliegendeWurst** [comment]
Ah, got it. Your suggestion is implemented in the latest commit. Feel free to install it using `cargo install --git 'https://github.com/FliegendeWurst/KIT-ILIAS-downloader`. I'll probably release a new version soon.

**Comment by funnym0nk3y** [comment]
Thank you very much! I'll try it soon.
